### PR TITLE
whoa

### PR DESCRIPTION
**Pull request by grzegorzj**

### CODE REVIEW

File: src/app/modules/lazy-image/lazy-image.component.ts [hunk 1]
```diff
@@ -170,26 +166,26 @@ export class LazyImageComponent implements AfterViewInit, OnInit {
...
+        if (this.stretchStrategy === 'original') {
             this.canvasHeight = this.imageHeight;
             this.canvasWidth = this.imageWidth;
         }
 
-        if (this.stretchStrategy === StretchStrategy.crop) {
-            this.stretchState = this.withinCropThreshold(this.imageWidth, this.imageHeight)
-                ? StretchStrategy.crop : StretchStrategy.stretch;
+        if (this.stretchStrategy === 'crop') {
+            this.stretchState = this.withinCropThreshold(this.imageWidth, this.imageWidth)
```
Comment (rokerkony) [inline src/app/modules/lazy-image/lazy-image.component.ts]: You didńt pull before all your. Hanges so my changes are gone

File: e2e/app.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { AppPage } from './app.po';
```
Comment (FlorianTopf) [inline e2e/app.e2e-spec.ts]: does it make much sense to keep e2e test configurations and the stub tests in this repo?

File: src/app/app.component.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'app';
```
Comment (FlorianTopf) [inline src/app/app.component.ts]: Linter doesn't cry about this?

File: src/app/modules/lazy-image/lazy-image.component.ts [hunk 4]
```diff
@@ -8,26 +8,23 @@ import {
...
     media: string;
     url: string;
 }
 
-export enum StretchStrategy {
-    crop = 'crop',
-    stretch = 'stretch',
-    original = 'original',
-}
+export type StretchStrategy = 'crop' | 'stretch' | 'original';
```
Comment (FlorianTopf) [inline src/app/modules/lazy-image/lazy-image.component.ts]: Any specific reason to move away from the enum?

File: e2e/app.e2e-spec.ts (hunk 2 above)
Comment (FlorianTopf) [reply e2e/app.e2e-spec.ts]: ... I'd either completely remove e2e testing stuff or just fix this one and make it very basic.

File: src/app/modules/lazy-image/lazy-image.component.ts (hunk 4 above)
Comment (grzegorzj) [reply src/app/modules/lazy-image/lazy-image.component.ts]: Yes. CLI doesn't support the new TypeScript.

**Pull request merged**